intSet/intSet/intSet.java

```diff
@@ -0,0 +1,190 @@
+        }
+    }
+    /**
+     * This method is used to check whether x is an element of the universal set or not
+     * @param x denotes the element which is to be checked
+     * @return true if element is present in the set else false
+     */
+    public boolean isMember(int x)
+    {
+        for(int i=0;i<arraySet.length;i++)
+        {
+            if(arraySet[i]==x)
+                return true;
+        }
+        return false;
+    }
+    /**
+     * Used to return the size of the set entered by the user
+     * @return the size of the set entered by the user
+     */
+    public int size()
+    {
+        return arraySet.length;
+    }
+    /**
+     * Used to check whether s is a subset of the set entered by the user
+     * @param s denotes the set which is to be checked that whether it is a subset or not
+     * @return true if it is a subset else return false
+     */
+    public boolean isSubSet(intSet s)
+    {
+        boolean flag=true;
+        for(int i=0;i<s.arraySet.length;i++)
+        {
+            flag=true;
+            for(int j=0;j<this.arraySet.length;j++)
+            {
+                if(s.arraySet[i]!=this.arraySet[j])
+                {
+                    flag=false;
+                }
+                else
+                {
+                    flag=true;
+                    break;
+                }
+            }
+            if(flag)
+            {
+                flag=true;
+            }
+            else
+                break;
+        }
+        return flag;
+    }
+    /**
+     * Used to calculate the complement of the inputed set by the user comparing it by the universal set
+     * @return the array consisting of the elements of the complement set 
+     */
+    public int[] getComplement()
+    {
+        int []complementArray=new int[universalSet.length-arraySet.length];
+        boolean flag=true;
+        int c=-1;
+        for(int i=0;i<universalSet.length;i++)
+        {
+            flag=false;
+            for(int j=0;j<arraySet.length;j++)
+            {
+                if(universalSet[i]!=arraySet[j])
+                {
+                    flag=true;
+                }
+                else 
+                {
+                    flag=false;
+                    break;
+                }
+            }
+            if(flag)
+                complementArray[++c]=universalSet[i];
+        }
+        return complementArray;
+    }
+    /**
+     * Used to calculate the union of two sets
+     * @param s1 denotes the first set
+     * @param s2 denotes the second set
+     * @return the array with elements of the union of both the subsets
+     */
+    public int[] union(intSet s1,intSet s2)
+    {
+        int count=0;
+        for(int i=0;i<s1.size();i++)
+        {
+            for(int j=0;j<s2.size();j++)
+            {
+                if(s1.arraySet[i]==s2.arraySet[j])
+                {
+                   count++;
+                }
+            }
+        }  

```

Comment:
This we are doing only for getting the count of the common elements of set so that we can create new array. So can't we use List here ?

Reply:
But, in the assignment it is written to do it with the help of an array